sdks/go/pkg/beam/core/runtime/exec/sdf.go

```diff
@@ -361,7 +371,7 @@ func (n *ProcessSizedElementsAndRestrictions) ProcessElement(_ context.Context,
 			n.rt = rt
 			n.elm = elm
 			n.SU <- n
-			err := n.PDo.processSingleWindow(&MainInput{Key: wElm, Values: mainIn.Values, RTracker: rt})
+			_, err := n.PDo.processSingleWindow(&MainInput{Key: wElm, Values: mainIn.Values, RTracker: rt})
```

Comment:
A comment explaining why we don't need to worry about ProcessContinuation here might be helpful

Reply:
This is a placeholder as we determined that an SDF shouldn't be allowed to explode windows. For now I just blank-assigned the field but in a later PR I believe we're going to remove that case altogether and return an error.

Reply:
SGTM

Reply:
Done